numba_dpex/core/targets/dpjit_target.py

```diff
@@ -28,6 +29,17 @@ class Dpex(CPU):
 dpex_function_registry = Registry()
 
 
+class DpexTypingContext(typing.Context):
+    """Custom typing context to support dpjit compilation."""
+
+    def load_additional_registries(self):
+        """Register the OpenCL API and math and other functions."""
```

Comment:
Incorrect docstring?

Reply:
The above comment is in the context of "OpenCL API", `dpjit` neither supports the functions in ocldecl registry, nor shout it.

Reply:
Yes, copy paste from kernel